badger_api/requests.py

```diff
@@ -31,34 +32,35 @@ def fetch_token_prices() -> Dict[str, float]:
     return prices
 
 
-def fetch_account_data(address: str):
+def fetch_claimable(page: int, chain: str):
     """
-    Fetch data from account data
-    :param address: address whose information is required
+    Fetch claimable data from account data
+    :param page: page to fetch data from
     """
-    data = (
-        requests.get(f"{badger_api}/accounts/{address}")
-        .json()
-        .get("claimableBalances", [])
-    )
+    data = requests.get(
+        f"{badger_api}/accounts/allClaimable?page={page}&chain={chain}"
+    ).json()
```

Comment:
What happens here if the api is down? We should gracefully catch error responses and handle them to prevent issues.